packaging/readme.md

```diff
@@ -51,3 +51,19 @@ curl localhost:9090/hello.html
 WARNING: the primary use case for this docker image is to have a quick'n'dirty skipper available quick. We don't
 necessarily update this image or the Dockerfile, so it may miss some important security updates. Use it at your
 own risk or, better, build your own image using these tools.
+
+```
+Copyright 2015 Zalando SE
```

Comment:
That's right according to our guidelines: https://github.com/zalando/zalando-howto-open-source#what-about-a-readme-note